docs/CHANGELOG.asciidoc

```diff
@@ -46,6 +46,8 @@
   (See {ml-pull}1238[#1238] and {pull}57254[#57254].)
 * Parallelize the feature importance calculation for classification and regression
   over trees. (See {ml-pull}1277[#1277].)
+* More realistic memory estimation requires lower memory limits than before
```

Comment:
I think it is worth mentioning classification and regression specifically since other analyses aren't changed:
```suggestion
* More realistic memory estimation for classification and regression means that these analyses will require
  lower memory limits than before.
```

Reply:
Updated